install/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+printf "\n\n######## deploy ########\n"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+oc project ${PROJECT} 2> /dev/null || oc new-project ${PROJECT}
+
+oc apply -k "${DIR}/istio-system/base"
+
+envsubst < "${DIR}/odh/overlays/default/kustomization_template.yaml" > "${DIR}/odh/overlays/default/kustomization.yaml"
```

Comment:
Should we use ` kustomize edit set namespace` or `vars` (https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/vars/) instead of this?

Reply:
Yes it should!  Thanks, didn't see that.  I'll have to script that out to use env vars.

Reply:
@vpavlin created an issue and will handle as a separate PR.